tests/driver_sx127x/README.md

```diff
@@ -0,0 +1,134 @@
+## About
+
+This is a manual test application for the SX127X radio driver.
+
+This test application uses the default pin configuration provided by the
+driver implementation and that matches the ST Nucleo 64 pins layout.
+It is best to use [SX1272](https://developer.mbed.org/components/SX1272MB2xAS/)
+or [SX1276](https://developer.mbed.org/components/SX1276MB1xAS/) mbed modules
+with nucleo boards or the all-in-one
+[ST P-NUCLEO-LRWAN1 LoRa kit](http://www.st.com/en/evaluation-tools/p-nucleo-lrwan1.html).
+
+If you have other hardware (boards, Semtech based LoRa module), you can adapt
+the configuration to your needs by updating the
+`drivers/sx127x/include/sx127x_params.h` file.
```

Comment:
Better don't teach new users bad practices. They can copy the file to the application directory and then update it ;-) This leaves the original untouched and prevents PRs with changed configs ;-).